.circleci/config.yml

```diff
@@ -31,7 +31,7 @@ jobs:
             wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
             /bin/bash Miniconda3-latest-Linux-x86_64.sh -b 
             cd /tmp && git clone https://github.com/expfactory/expfactory.git
-            cd /tmp/expfactory && $HOME/miniconda3/bin/python setup.py install
+            cd /tmp/expfactory && $HOME/miniconda3/bin/pip install .
```

Comment:
hey @JuneOh710 ! So just see this change here - when we merge you can update your other PR from main to also have this change and the install should work.